components/Navbar/Navbar.test.tsx

```
@@ -0,0 +1,87 @@
+  TripsListElement: () => <div data-testid="trips-list-element">TripsListElement</div>,
+}));
+
+jest.mock('@/components/Welcome/Welcome', () => ({
+  Welcome: () => <div data-testid="welcome">Welcome</div>,
+}));
+
+jest.mock('@/components/StravaLoginButton/StravaLoginButton', () => ({
+  StravaLoginButton: ({ onAuthCode }: { onAuthCode: (code: string) => void }) => (
+    <button data-testid="strava-login-btn" onClick={() => onAuthCode('code')}>
+      Connect with Strava
+    </button>
```

ESLint

[error] Missing an explicit type attribute for button

(react/button-has-type)